src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  padding: 0;
+
+  &__star {
+    display: inline;
+    width: $star-size;
+    height: $star-size;
```

Comment:
Also would be nice to create variable from it, to reuse if we need to make square form something